docs/deploy-full-rp-service-in-dev.md

```diff
@@ -29,11 +29,42 @@
     make dev-config.yaml
     ```
 
-
 1. Run `make deploy`
-    > __NOTE:__ This will fail on the first attempt to run due to certificate and container mirroring requirements.
+    > __NOTE:__ This will fail on the first attempt to run due to certificate or AKS requirements.
+
+    > __NOTE:__ If the deployment fails with `InvalidResourceReference` due to the RP Network Security Groups not found, delete the "gateway-production-predeploy" deployment in the gateway resource group, and re-run `make deploy`.
+
+    > __NOTE:__ If the deployment fails with `A vault with the same name already exists in deleted state`, then you will need to recover the deleted keyvaults from a previous deploy using: `az keyvault recover --name <KEYVAULT_NAME>` for each keyvault, and re-run.
+
+1. Deploy AKS by running these commands from the ARO-RP root directory:
+    ```bash
+    source ./hack/devtools/deploy-shared-env.sh
+    deploy_aks_dev
+    ```
+    > __NOTE:__ If the AKS deployment fails with missing RP VNETs, delete the "gateway-production-predeploy" deployment in the gateway resource group, and re-run `make deploy` and then re-run `deploy_aks_dev`.
 
-    > __NOTE:__ If the deployment fails with `InvalidResourceReference` due to the RP Network Security Groups not found, delete the gateway predeploy deployment, and re-run.
+1. Install Hive into AKS
+    1. Deploy a VPN Gateway and in order to be able to connect to AKS from your local machine:
+        ```bash
+        source ./hack/devtools/deploy-shared-env.sh
+        deploy_vpn_for_dedicated_rp
+        ```
```

Comment:
Hey David, it looks like `deploy_vpn_for_dedicated_rp` needs to be ran before `deploy_aks_dev`.
```
[tschneid@fedora ARO-RP]$ deploy_aks_dev
########## Deploying aks-development in RG tschneid-aro-eastus ##########
ERROR: {"status":"Failed","error":{"code":"DeploymentFailed","message":"At least one resource deployment operation failed. Please list deployment operations for details. Please see https://aka.ms/DeployOperations for usage details.","details":[{"code":"BadRequest","message":"{\r\n  \"error\": {\r\n    \"code\": \"InvalidGlobalResourceReference\",\r\n    \"message\": \"Resource /subscriptions/fe16a035-e540-4ab7-80d9-373fa9a3d6ae/resourceGroups/tschneid-aro-eastus/providers/Microsoft.Network/virtualNetworks/dev-vpn-vnet referenced by resource /subscriptions/fe16a035-e540-4ab7-80d9-373fa9a3d6ae/resourceGroups/tschneid-aro-eastus/providers/Microsoft.Network/virtualNetworks/aks-net/virtualNetworkPeerings/peering-dev-vpn-vnet was not found. Please make sure that the referenced resource exists.\",\r\n    \"details\": []\r\n  }\r\n}"},{"code":"NotFound","message":"{\r\n  \"error\": {\r\n    \"code\": \"ResourceNotFound\",\r\n    \"message\": \"The Resource 'Microsoft.Network/virtualNetworks/dev-vpn-vnet' under resource group 'tschneid-aro-eastus' was not found. For more details please go to https://aka.ms/ARMResourceNotFoundFix\"\r\n  }\r\n}"}]}}
```

Reply:
Thanks, I changed it around.